### PR TITLE
ci: set whether it is the latest release by using 'ncipollo/release-action and update install.sh

### PR DESCRIPTION
**Pull request by zyy17**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

1. Set whether it is the latest release by using 'ncipollo/release-action';
2. Modify greptimedb install script to use the latest nightly version binary;

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by sunng87 — APPROVED** [review]
:rocket:

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -319,27 +319,43 @@ jobs:
           SCHEDULED_BUILD_VERSION=${{ env.SCHEDULED_BUILD_VERSION_PREFIX }}-${{ env.SCHEDULED_PERIOD }}-$buildTime
           echo "SCHEDULED_BUILD_VERSION=${SCHEDULED_BUILD_VERSION}" >> $GITHUB_ENV
 
+      # Only publish release when the release tag is like v1.0.0, v1.0.1, v1.0.2, etc.
+      - name: Set whether it is the latest release
+        run: |
+          if [[ "${{ github.ref_name }}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "prerelease=false" >> $GITHUB_ENV
+            echo "makeLatest=true" >> $GITHUB_ENV
```
Comment (v0y4g3r) [inline .github/workflows/release.yml:327]: Can we only keep `makeLatest` since `prerelease` === `!makeLatest`

Comment (zyy17) [reply .github/workflows/release.yml:327]: Maybe it's difficult to describe in GitHub action yaml, and it needs the true boolean variable support.

**Review by v0y4g3r — APPROVED** [review]
LGTM

**Pull request merged**